RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.0.7
+=====
+* Update instructions of translation process #381
```

Comment:
Let's remove this one, as the change was made in the readme and not in the app.

Reply:
This has been addressed in https://github.com/Automattic/studio/pull/412/commits/be5c825387362755911d8fb8897b32a8decc8165.